web_src/css/repo.css

```diff
@@ -2128,14 +2128,16 @@
 }
 
 #search-user-box .results .result .image {
-  float: left;
-  margin-right: 8px;
+  order: 0;
+  margin-right: 12px;
   width: 2em;
   height: 2em;
+  min-width: 2em;
+  min-height: 2em;
 }
 
 #search-user-box .results .result .content {
-  margin: 6px 0; /* this trick is used to align with the sibling avatar image */
+  margin: 6px 0;
```

Comment:
According to my test, I think it should be:

```
.ui.search > .results .result .image + .content {
    margin: 0; /* reset Fomantic's margin because we have `align-items: center;` on the `.result` parent
}
```

Reply:
Not quite, correct comment in https://github.com/go-gitea/gitea/pull/29002/commits/6da06851c76a0358c4063d58f2e0cedb283000b1.